deployments/operator/otel-tracing/sitecustomize.py

```
@@ -0,0 +1,26 @@
+
+    # Initialize Traceloop with environment variables
+    Traceloop.init(
+        telemetry_enabled=os.getenv("AMP_TRACELOOP_TELEMETRY_ENABLED", "false"),
+        app_name=os.getenv("AMP_APP_NAME", "sample-app"),
+        resource_attributes={"env": os.getenv("AMP_ENV", "prod"), "version": os.getenv("AMP_APP_VERSION", "1.0.0")},
+        api_endpoint=os.getenv("AMP_OTEL_EXPORTER_OTLP_ENDPOINT", "data-prepper.openchoreo-observability-plane.svc.cluster.local:21893"),
+        headers={"x-api-key": os.getenv("AMP_API_KEY", "")}
+    )
+    Traceloop.set_association_properties({ "component_id": os.getenv("AMP_COMPONENT_ID", "default-component") })
+    logger.info("Automatic Tracing initialized successfully.")
+except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)